src/langchain_google_firestore/document_converter.py

```diff
@@ -15,24 +15,26 @@
 from __future__ import annotations
 
 import json
-from typing import TYPE_CHECKING, Any, List
+from typing import TYPE_CHECKING, Any, List, Optional
 
 from google.cloud.firestore import DocumentReference, GeoPoint  # type: ignore
+from google.cloud.firestore_v1.vector import Vector  # type: ignore
 from langchain_core.documents import Document
 
 if TYPE_CHECKING:
-    from google.cloud.firestore import Client, DocumentReference, DocumentSnapshot
+    from google.cloud.firestore import Client, DocumentSnapshot
 
 
 FIRESTORE_TYPE = "firestore_type"
 DOC_REF = "document_reference"
 GEOPOINT = "geopoint"
+VECTOR = "vector"
 
 
 def convert_firestore_document(
     document: DocumentSnapshot,
-    page_content_fields: List[str] = [],
```

Comment:
I checked where it's referenced, I think we won't need code changes since it's already checked with `or`
```
set_page_fields = set(
        page_content_fields or (data_doc.keys() - set(metadata_fields or []))
    )
```

Reply:
mypy is passing 🎉